test-files/article.html

```diff
@@ -108,9 +108,9 @@ <h3 class="code-title">Code</h3>
 			<h2 class="test-title" id="ar-006">Test ar-006: Article with <code>aria-describedby</code></h2>
 
 			<div class="test-case">
-				<article aria-describedby="ar-006-describedby">
-					<h1 id="ar-006-describedby">h1 referenced by aria-describedby</h1>
-					<p>Aria-describedby should be used as accessible description.</p>
+				<article aria-describedby="ar-006-describedby6">
+					<h1 id="ar-006-describedby6">h1 referenced by aria-describedby6</h1>
```

Comment:
Do these need the 6 added to the end of the ID when they already have the unique ar-006- prefixing the IDs?